helper/completion_generator.py

```diff
@@ -248,6 +248,4 @@ def traverse(self, tree=None, level=0, origin=None):
 
 complete -F _kiwi -o default kiwi
 complete -F _kiwi -o default kiwi-ng
-complete -F _kiwi -o default kiwi-ng-2
-complete -F _kiwi -o default kiwi-ng-3
```

Comment:
This file still exists, so you probably want to keep it, especially as other distros may not necessarily create the symlinks..

Reply:
Hmm, ok I revert it

Reply:
You can drop `kiwi-ng-2` entry, though.

Reply:
yes will do that in another commit, just a sec